src/transformState.js

```diff
@@ -5,7 +5,36 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    const operation = action.type;
+
+    switch (operation) {
```

Comment:
Use a destructurization here for action
```suggestion
  for (const action of actions) {
    const {} = action;

    switch (type) {
```